### ISSUE TITLE
Confirm processing of QLFS/NEET/CPI/vacancies data

### ISSUE THREAD

**Issue by taz-chiles** [open-innovations/yff-data#74]
(Edited by Michelle)

Email Chris, Andrea & Zach to confirm how we are processing data for each of these. To include (although other things may be added):

- [ ] QLFS data - and whether we are discarding latest relevant data
- [ ] Definition of long-term unemployment (6 month or 12 month)
- [ ] Possible also pick up questions about GDP?  #34 

When emailing, may also want to include #82 & #83

**Comment by gilesdring** [comment]
Write blog post about this and share that.

**Comment by gilesdring** [comment]
To expedite this, I will start with an email to YFF, which once we have some confirmation I will write up as a blog.

**Comment by gilesdring** [comment]
Email sent end of last week. Awaiting response.

**Comment by gilesdring** [comment]
Text of email:

The QLFS pipeline is documented on the data site https://data.youthfuturesfoundation.org/dashboard/qlfs/pipeline/.
 
QLFS (A06) data currently focusses on people not in full-time education, and is not yet split by gender.
 
The quarterly processing (discarding two out of three months) was as described to us by YFF contacts, and we believe this is the regular processing that takes place on this data. We made the decision to align the quarters with the NEET stats during discussion with YFF. Each of these decisions is easy to reverse in the pipeline.
 
We discussed including the pre-pandemic period as a benchmark. We could either extend to 4 years processing (which would include 2019, for now), or pin the start of the extract to Jan 2019, so we end up with an increasing date range.
 
For the UNEM01 data, we are creating an aggregated 16-24 years figure, which is not published by ONS. We currently use the over 12-months figures to determine long-term unemployment. This is simple to switch to 6 months if desired.
 
NEET processing is similarly documented at https://data.youthfuturesfoundation.org/dashboard/neet/pipeline/, although note we also process the separate gender information here.
 
We will ultimately write up the processing of the economic indicators as well. A present we're working on pipelines for CPI and vacancies data. We discussed the appropriate set of queries to use to create a CPI stat meaningful for the work of YFF. I'd be interested you hear your thoughts about this.
 
There has also been some discussion about inclusion of GDP in this economic dashboard. We weren't able to chat about this during the session, but again: it'd be good to hear your view of the importance of this as a measure.
